packages/upswyng-web/src/components/BackButton.tsx

```diff
@@ -1,7 +1,8 @@
 import { Icon, Typography } from "@material-ui/core";
+import React, { useState } from "react";
 
 import { ArrowBack } from "@material-ui/icons";
-import React from "react";
+import { Redirect } from "react-router";
```

Comment:
Instead of using a rendered `<Redirect/>` React router actually has a [`useHistory` hook](https://reacttraining.com/blog/react-router-v5-1/#usehistory) that will allow you to use `history.push()` instead.

Your implementation definitely works, but I think we should add to history rather than replace the current location as `<Redirect/>` does.

Reply:
hook master